docs/immortal-empires/slaanesh.md

```diff
@@ -0,0 +1,33 @@
+# Slaanesh 
+
+## N'Kari - The Arch-Temptor
+
+* **Route I - The Great Game**

```

Comment:
I wanted to use "The Great Game" as per the `Achieve the highest corruption in a minimum of 100 regions`, as we alreayd have code developed for that. However, maybe we want a name more suited for the Ulthuan thing

Reply:
I modified the flavour text to mention Ulthuan. It should be fine.

I put conquer Ulthuan in here because they are mechanically the same goal. You unlock the great game levels by expanding your territory/vassal's territory.